HISTORY.rst

```diff
@@ -5,6 +5,10 @@ History
 Unreleased
 ----------
 
+* New glinter warnings were added (see bug 1682980):
+  - `SHORT_NAME`: When a category name, metric name or ping name is less than 3 characters.
```

Comment:
We have `glean.error.io` now.
And also `glean.internal.metrics.os`.
The last part is the name (`io` or `os`).
Will we need to disable this lint on them then?

Reply:
Hm. Maybe we should restructure the limits so that they don't apply if the category+name length is at least six? I knew this was going to be a bikeshed of epic colour : )

(( Also, why is it `os` and not `os_name`? Clearly I wasn't invited to the bikeshed painting party then. ))